doc/specs/vulkan/chapters/VK_EXT_debug_report.txt

```diff
@@ -111,24 +113,28 @@ include::../api/funcpointers/PFN_vkDebugReportCallbackEXT.txt[]
     type of object being used or created at the time the event was
     triggered.
   * pname:object gives the object where the issue was detected.
-    pname:object may be ename:VK_NULL_HANDLE if there is no object
+    pname:object can: be ename:VK_NULL_HANDLE if there is no object
```

Comment:
Wrong spot. This one should probably be "may:" (because it is basically output from Vulkan).